src/plugins/hotkeys/hotkeys.ts

```diff
@@ -53,42 +53,27 @@ export default class Hotkeys {
 	 * @param e Event listener datas
 	 */
 	onKeydown(e: KeyboardEvent) {
-		const activeElement = document.activeElement
 		const { keyCode } = e
-
-		if (
-			[9, 32, 37, 39].includes(keyCode) &&
-			this.player.Vlitejs.autoHideGranted &&
```

Comment:
Why do you remove this line? It is a user option

Reply:
You're right, I restored it in a commit